### PR TITLE
Feat: determine certificates for reissuance (EXPOSUREAPP-11992)

### PR DESCRIPTION
**Pull request by mlenkeit**
This PR adds logic to the DCC Wallet Info so that it also determines certificates that are eligible for reissuance.

### CODE REVIEW

**Review by hermesmar — APPROVED** [review]

File: resources/i18n/de/strings.xml [hunk 1]
```diff
@@ -59,4 +59,11 @@
   <!-- XTXT: Booster Notification - Card Title -->
   <string name="BOOSTER_NOTIFICATION_SUBTITLE_TEXT">"auf Grundlage Ihrer gespeicherten Zertifikate"</string>
 
+  <!-- XHED: DCC Reissuance - Card Title -->
+  <string name="DCC_REISSUANCE_TITLE_TEXT">"Zertifikat aktualisieren"</string>
+  <!-- XTXT: DCC Reissuance - Card Subtitle -->
+  <string name="DCC_REISSUANCE_SUBTITLE_TEXT">"Neuausstellung direkt über die App vornehmen."</string>
+  <!-- XTXT: DCC Reissuance - Card Long Text -->
+  <string name="DCC_REISSUANCE_LONG_TEXT">"Die Spezifikationen der EU für Booster-Impfzertifikate wurden geändert. Dieses Zertifikat entspricht nicht den aktuellen Spezifikationen. Das Impfzertifikat ist zwar weiterhin gültig, es kann jedoch sein, dass bei einer Prüfung die Booster-Impfung nicht erkannt wird. Bitte lassen Sie sich daher ein neues Impfzertifikat ausstellen.\n\nSie können ein neues Impfzertifikat direkt kostenlos über die App anfordern. Hierfür ist Ihr Einverständnis erforderlich."</string>
```
Comment (mlenkeit) [inline resources/i18n/de/strings.xml:67]: This text uses _Booster_ although we usually use _Auffrischimpfung_, open for discussion

File: test/unit/ccl/functions/__analyzeDccWallet.spec.js [hunk 2]
```diff
@@ -193,6 +195,56 @@ End of debugging: ${chalk.magenta(testCaseDescription)}`
...
+
+            context('certificateReissuance', () => {
+              const {
+                certificateReissuance: expCertificateReissuance
+              } = assertions
+
+              has('certificateReissuance') &&
+              it('check certificateReissuance', () => {
+                if (expCertificateReissuance) {
+                  expect(output).to.have.nested.property('certificateReissuance')
```
Comment (T-Bendrath) [inline test/unit/ccl/functions/__analyzeDccWallet.spec.js:207]: I see no need for the `.nested` chain occurrences when testing the `certificateReissuance` node. Removing it won't change the test result.

**Review by T-Bendrath — APPROVED** [review]

File: test/unit/ccl/functions/__analyzeDccWallet.spec.js (hunk 2 above)
Comment (mlenkeit) [reply test/unit/ccl/functions/__analyzeDccWallet.spec.js]: see https://github.com/corona-warn-app/cwa-app-ccl/pull/32/commits/0a9e5e449359c794c2c05472cd07ee01c80d9865

File: test/unit/ccl/functions/getDccWalletInfo.spec.js [hunk 3]
```diff
@@ -389,6 +389,82 @@ End of debugging: ${chalk.magenta(testCaseDescription)}`
...
+                const {
+                  certificateReissuance: expCertificateReissuance
+                } = expWalletInfo
+
+                has('certificateReissuance') &&
+                it('check certificateReissuance', () => {
+                  if (expCertificateReissuance) {
+                    expect(output).to.have.nested.property('certificateReissuance')
+                  } else {
+                    expect(output).not.to.have.nested.property('certificateReissuance')
```
Comment (T-Bendrath) [inline test/unit/ccl/functions/getDccWalletInfo.spec.js:403]: No need for the `.nested` chain occurrences when testing the `certificateReissuance` node. Similar to the changes in https://github.com/corona-warn-app/cwa-app-ccl/pull/32/commits/0a9e5e449359c794c2c05472cd07ee01c80d9865

**Review by CV113 — APPROVED** [review]
UA approved

**Pull request merged**